agents/go-agents/terminal-agent/main.go

```diff
@@ -26,13 +26,16 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"regexp"
 	"time"
 
+	"strconv"
```

Comment:
the same as here https://github.com/eclipse/che/pull/8088#discussion_r159417582

Reply:
good catch!